client/packages/common/src/intl/utils/DateUtils.ts

```diff
@@ -33,13 +33,17 @@ import {
   previousMonday,
   endOfWeek,
   setMilliseconds,
+  addMilliseconds,
 } from 'date-fns';
+import { getTimezoneOffset } from 'date-fns-tz';
 
 export const MINIMUM_EXPIRY_MONTHS = 3;
 
 const dateInputHandler = (date: Date | string | number): Date => {
+  // console.log('date input handler', date);
+  // TODO this is being called for every patient in list (it shouldn't)
```

Comment:
This TODO doesn't make sense in a utils method... Should these two lines be removed?